jwst/assign_wcs/nircam.py

```diff
@@ -410,15 +406,43 @@ def wfss(input_model, reference_files):
 
     # pass the x0,y0, wave, order, through the pipeline
     imagepipe = []
-    world = image_pipeline.pop()
+    world = image_pipeline.pop()[0]
```

Comment:
Wouldn't this cause `DeprecationWarning`?
```suggestion
    world = image_pipeline.pop().frame
```

Reply:
Ah, after the `pop` it's just a tuple.

Reply:
That's unexpected (to me)